### DISCUSSION TITLE
agency-aviary@nasa.gov not working

### DISCUSSION THREAD

**Discussion by atymes** [OpenMDAO/Aviary#476] [General]
I tried sending an email to agency-aviary@nasa.gov and got this error message from postmaster@nasa.gov:

Your message to agency-aviary@nasa.gov couldn't be delivered.
agency-aviary wasn't found at nasa.gov.

Is this a temporary issue or is the address disabled?

Is it appropriate to post here, a message that would have been sent to that address?

**Comment by jkirk5** [comment]
Thanks for reporting - I've been able to confirm that the email domain indeed seems no longer valid, and will investigate why that suddenly happened. Feel free to post questions about Aviary here in the discussion board!

**Comment by jkirk5** [comment]
Try agency-aviary@mail.nasa.gov
It looks like there may have been an unfortunate typo on the promotional material

**Reply by atymes** [reply]
Alright, I have resent my email there.  It did not immediately bounce like the other address did, so hopefully that email got through.  Although, it's not just your presentation; other posted material such as https://sacd.larc.nasa.gov/aviary-takes-flight/ points to agency-aviary@nasa.gov .

So you know, this email is actually intended for you, assuming you are Jason Kirk.  I didn't get your direct email address, so I'm using this as a way to route a message to you.  Though, the message is also somewhat intended for the Aviary team at NASA too.  It has to do with a potential funding opportunity for Aviary, thus my preference to discuss via email rather than here.

**Reply by jkirk5** [reply]
Yep, got the email! And yeah, this is Jason

**Reply by atymes** [reply]
Then I will await your response to the email.  Thank you!
